modules/actions/workflows.go

```diff
@@ -85,138 +83,178 @@ func DetectWorkflows(commit *git.Commit, triggedEvent webhook_module.HookEventTy
 }
 
 func detectMatched(commit *git.Commit, triggedEvent webhook_module.HookEventType, payload api.Payloader, evt *jobparser.Event) bool {
+	if convertFromGithubEvent(evt) != triggedEvent.Event() {
+		return false
+	}
+
+	// with no special filter parameters
 	if len(evt.Acts) == 0 {
 		return true
 	}
 
 	switch triggedEvent {
-	case webhook_module.HookEventCreate:
-		fallthrough
-	case webhook_module.HookEventDelete:
-		fallthrough
-	case webhook_module.HookEventFork:
+	case webhook_module.HookEventCreate,
+		webhook_module.HookEventDelete,
+		webhook_module.HookEventFork,
```

Comment:
These three events returned false and now true, is that correct?

Reply:
I think yes. Because I have checked whether the name is the same in lines 86-88.